test/integration/webhook/v1alpha1/queue_test.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+	"k8s.io/apimachinery/pkg/api/errors"
+
+	"sigs.k8s.io/kueue/pkg/util/testing"
+)
+
+const queueName = "queue-test"
+
+var _ = ginkgo.Describe("Queue validating webhook", func() {
+	ginkgo.Context("When updating a Queue", func() {
```

Comment:
Add a normal path as you do in workload validation?

Reply:
didn't we settle on using when instead of context?